mllib/src/test/scala/org/apache/spark/ml/classification/ProbabilisticClassifierSuite.scala

```diff
@@ -56,6 +56,21 @@ class ProbabilisticClassifierSuite extends SparkFunSuite {
     val testModel = new TestProbabilisticClassificationModel("myuid", 2, 2)
     assert(testModel.friendlyPredict(Vectors.dense(Array(1.0, 2.0))) === 1.0)
   }
+
+  test("test tiebreak") {
+    val testModel = new TestProbabilisticClassificationModel("myuid", 2, 2)
+      .setThresholds(Array(0.4, 0.4))
+    assert(testModel.friendlyPredict(Vectors.dense(Array(0.6, 0.6))) === 0.0)
+  }
+
+  test("bad thresholds") {
+    intercept[IllegalArgumentException] {
```

Comment:
Honestly I don't really see the theoretical justification even for the `cutoff` approach - a hard threshold works in the binary case but not really in multi-class. Anything related to thresholds I've seen is mostly related to OneVsRest scenarios where the threshold can be adjusted per class...

So yes my point here was that if we are just copying the approach from the R package, let's be consistent with it.

But yes, the main issue is ensuring they're all positive, so I'm ok with removing that constraint.